src/test/java/org/purescript/psi/PSImportedClassTest.kt

```diff
@@ -0,0 +1,22 @@
+package org.purescript.psi
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import junit.framework.TestCase
+import org.purescript.file.PSFile
+
+
+class PSImportedClassTest : BasePlatformTestCase() {
+
+    fun `test imported class has correct name`() {
+        val file = myFixture.configureByText(
+            "Foo.purs",
+            """
+                module Foo where
+                import Bar (class Qux)
+            """.trimIndent()
+        ) as PSFile
+        val importDecl = file.module.importDeclarations.single()
+        val importedClass = importDecl.importList!!.importedItems.single() as PSImportedClass
+        TestCase.assertEquals("Qux", importedClass.name)
+    }
+}
```

Comment:
There are many tests that are similar to this, but I feel like they all should exist. I put them in different classes because I think it's lazy to put them all in e.g. `PSImportedItemTest`. We will likely have need for separate test classes for each type of imported item eventually.